docs/docs/behaviors/mouse-emulation.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Mouse Emulation Behaviors
+sidebar_label: Mouse Emulation
+---
+
+## Summary
+
+Mouse emulation behaviors send mouse movements, button presses or wheel actions.
+
+Please view [`dt-bindings/zmk/mouse.h`](https://github.com/zmkfirmware/zmk/blob/main/app/include/dt-bindings/zmk/mouse.h) for a comprehensive list of signals.
+
+## Keycode Defines
+
+To make it easier to encode the HID keycode numeric values, most keymaps include
+the [`dt-bindings/zmk/mouse.h`](https://github.com/zmkfirmware/zmk/blob/main/app/include/dt-bindings/zmk/mouse.h) header
+provided by ZMK near the top:
+
+```
+#include <dt-bindings/zmk/mouse.h>
+```
+
+Doing so allows using a set of defines such as `MOVE_UP`, `MOVE_DOWN`, `LCLK` and `WHEEL_UP` with these behaviors.
+
+## Mouse Button Press
+
+This behavior can press/release up to 16 mouse buttons.
+
+### Behavior Binding
+
+- Reference: `&mkp`
+- Parameter: A `uint16` with each bit referring to a button.
+
+Example:
+
+```
+&mkp LCLK
+```
+
+## Mouse Movement
+
+This behavior is used to manipulate the cursor.
+
+### Behavior Binding
+
+- Reference: `&mmv`
+- Parameter: A `uint32` with the first 16 bits relating to horizontal movement
```

Comment:
If you give the examples with macros, you don't have to explain the bit packing.

Reply:
I'll make a note to refactor the docs once the macro format is settled.